week-4/InClass/E-arrays-of-objects/exercise-2.js

```diff
@@ -40,11 +40,25 @@ WRITE YOUR CODE BELOW
 */
 
 
-let destinationNamesWithin500Kms = // Complete here
+let destinationNamesWithin500Kms = travelDestinations
+.filter(destination=>destination.distanceKms <500)  // Complete here
+.map(destination=>destination.destinationName);
 
-let destinationNameReachableByFerry = // Complete here
+console.log(destinationNamesWithin500Kms);
 
-let destinationNamesMoreThan300KmsAwayByTrain = // Complete here (PRINT THE RESULT IN THE CONSOLE USING FOREACH)
+let destinationNameReachableByFerry = travelDestinations
```

Comment:
It looks like you didn't quite finish off this file - can you give it another go? We're always around on slack to talk things through if you need a hand :)